storey/targets.py

```diff
@@ -819,13 +816,9 @@ def __init__(
         table_col: Optional[str] = None,
         supertable: Optional[str] = None,
         tag_cols: Union[str, List[str], None] = None,
-        timeout: Optional[int] = None,
         time_format: Optional[str] = None,
         **kwargs,
     ):
```

Comment:
why did you remove the check that it's taosws?

Reply:
it's only relevant to REST connection, there is no timeout param with the websocket connection